app/positions/components/onboarding/SetupPositions.tsx

```
@@ -0,0 +1,471 @@
+    const newLockedAmounts = new Set(lockedAmounts);
+    if (lockedAmounts.has(marketKey)) {
+      newLockedAmounts.delete(marketKey);
+    } else {
+      newLockedAmounts.add(marketKey);
+    }
+    setLockedAmounts(newLockedAmounts);
+  }, [lockedAmounts]);
+
+  const handlePercentageChange = useCallback((marketKey: string, newPercentage: number) => {
+    // If the input is invalid (NaN), set it to 0
+    if (isNaN(newPercentage)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)